### PR TITLE
@ima/devtools

### PR DESCRIPTION
**Pull request by jsimck**

### CODE REVIEW

File: packages/devtools-scripts/rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      include: 'node_modules/**'
+    })
+  ]
+};
+
+if (env === 'es' || env === 'cjs') {
+  config.output = Object.assign(config.output || {}, { format: env });
+}
+
+if (env === 'umd') {
```
Comment (mjancarik) [inline packages/devtools-scripts/rollup.config.js]: Remove useless condition

File: packages/devtools-scripts/src/main.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+const ImaMainModules = [
+  'onLoad',
+  'getInitialImaConfigFunctions',
+  'getNamespace',
+  'getInitialPluginConfig',
+  'createImaApp',
+  'getClientBootConfig',
+  'bootClientApp',
+  'routeClientApp',
+  'hotReloadClientApp',
```
Comment (mjancarik) [inline packages/devtools-scripts/src/main.js]: The key hotReloadClientApp was removed from IMA@17

File: packages/devtools-scripts/rollup.config.js (hunk 1 above)
Comment (jsimck) [reply packages/devtools-scripts/rollup.config.js]: Removed.

File: packages/devtools-scripts/src/main.js (hunk 2 above)
Comment (jsimck) [reply packages/devtools-scripts/src/main.js]: Fixed.

**Pull request merged**